.travis/trigger_browser_tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+BODY='{"request": {"branch": "master"}}'
+
+curl -s -X POST \
+   -H "Content-Type: application/json" \
+   -H "Accept: application/json" \
+   -H "Travis-API-Version: 3" \
+   -H "Authorization: token $QUARCKSTER_TOKEN" \
```

Comment:
Do you need us to set this token on travis?  I see it has been set, but it needs to be set by somebody on the PF team so we can keep track of what is used.  @karelhala @quarckster

Reply:
Yes, please set up this token in Travis settings. I've already provided token to @karelhala.